### PR TITLE
EPMRPP-64279: change tryDemo to askPricing button on mobile view

### PR DESCRIPTION
**Pull request by renkyoji**

### CODE REVIEW

File: src/views/components/modals/askPricingModal/index.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ export default Epoxy.View.extend({
     BaronScroll($('[data-js-content]', this.$el));
   },
   resize() {
-    if ($(document).width() <= 767) {
+    if ($(document).width() <= 1) {
       this.destroy();
     }
   },
```
Comment (chivekrodis) [inline src/views/components/modals/askPricingModal/index.js]: Does it method is necessary?

**Review by chivekrodis — CHANGES_REQUESTED** [review]

File: src/views/components/modals/askPricingModal/index.js (hunk 1 above)
Comment (renkyoji) [reply src/views/components/modals/askPricingModal/index.js]: if delete this method, modal will not close when click outside

Comment (chivekrodis) [reply src/views/components/modals/askPricingModal/index.js]: The `resize` method doesn't implement an any logic to close a modal by clicking on the background.
In my opinion you should remove `resize` method, in all places where it is used and check it again.
Also, remove unnecessary import `selectize`.
`onDestroy` unnecessary too.

File: src/views/pages/index-page/index-page_welcome/index.js [hunk 2]
```diff
@@ -37,4 +39,8 @@ export default IndexPageSection.extend({
     e.preventDefault();
     Router.modals.show(new PreviewModal({ src: $(e.currentTarget).attr('href') }));
   },
+  onClickAskPricing(e) {
+    e.preventDefault();
+    Router.modals.show(new AskPricingModal());
```
Comment (chivekrodis) [inline src/views/pages/index-page/index-page_welcome/index.js:44]: Please, make sure that modal window is display correctly on mobile view up to 375px

**Review by chivekrodis — CHANGES_REQUESTED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**